mulighetsrommet-arena-adapter/src/main/kotlin/no/nav/mulighetsrommet/arena/adapter/routes/ApiRoutes.kt

```diff
@@ -17,7 +18,23 @@ fun Route.apiRoutes() {
         val request = call.receive<ReplayTopicEventsRequest>()
 
         JobRunners.executeBackgroundJob {
-            arenaEventService.replayEvents(table = request.table)
+            arenaEventService.replayEvents(
+                table = request.table,
+                status = request.status?.let { ArenaEvent.ConsumptionStatus.valueOf(it) }
+            )
+        }
+
+        call.respond(HttpStatusCode.Created)
+    }
+
+    put("api/event/replay") {
+        val request = call.receive<ReplayTopicEventRequest>()
+
+        JobRunners.executeBackgroundJob {
+            arenaEventService.replayEvent(
```

Comment:
Siden dette kun er for et enkelt event så trenger vi ikke å wrappe det med `JobRunners.executeBackgroundJob`